brion/spikeReport.h

```diff
@@ -53,7 +53,7 @@ class SpikeReport;
  *
  * This class is not thread-safe except where stated otherwise.
  */
-class SpikeReport : public boost::noncopyable
+class SpikeReport
```

Comment:
I get this point, but sometimes =delete is used for some but not all constructors/assignment operators in order. In this case you will have to check all of them to check if the type is noncopyable or not. For me, boost::noncopyable is much clearer and makes the code shorter.